Misc/NEWS

```diff
@@ -303,6 +303,10 @@ Extension Modules
 Library
 -------
 
+- bpo-30017: Allowed calling the close() method of the zip entry writer object
+  multiple times.  Writing to closed zip entry writer object now always produce
+  a ValueError.
```

Comment:
Needs singular form of “produces”. I would put “Writing to a closed writer now always produces a ValueError.”